src/esw/brushed_motors/ROSHandler.cpp

```diff
@@ -57,27 +73,82 @@ std::optional<float> ROSHandler::moveControllerOpenLoop(const std::string& name,
     return std::make_optional<float>(controller->getCurrentAngle());
 }
 
+// REQUIRES: nothing
+// MODIFIES: nothing
+// EFFECTS: Moves a controller in closed loop.
+std::optional<float> ROSHandler::moveControllerClosedLoop(const std::string& name, float targetAngle) {
+    auto controller_iter = ControllerMap::controllersByName.find(name);
+
+    if (controller_iter == ControllerMap::controllersByName.end()) {
+        return std::nullopt;
+    }
+
+    // TODO: actually write this code idk
```

Comment:
Remove this todo? Or is this still valid lol

Reply:
This is still valid - closed loop is being implemented [here](https://github.com/umrover/mrover-ros/tree/ed/close-loop).